src/extension/app/components/plugin/plugin-action-bar.js

```diff
@@ -257,11 +276,42 @@ export class PluginActionBar extends ConnectedElement {
     }
 
     const properties = html`
-      <sp-action-button id="properties" quiet>
+      <sp-action-menu id="sidekick-menu" placement="top" quiet>
         <sp-icon slot="icon" size="l">
-          ${ICONS.PROPERTIES}
+          ${ICONS.HAMBURGER_ICON}
         </sp-icon>
-      </sp-action-button>`;
+        <sp-menu-group>
+          ${siteStore.transient ? html`
+              <sp-menu-item class="icon-item add-project" value="projectadded" @click=${this.handleItemSelection}>
+                <sp-icon slot="icon" size="m">
+                  ${ICONS.PLUS_ICON}
+                </sp-icon>
+                Add Project
```

Comment:
Move menu labels to `messages.json` and use localized strings

Reply:
Whoops, thanks for catching